features/step_definitions/log_in_steps.rb

```diff
@@ -0,0 +1,23 @@
+And(/^I click "([^"]*)" button$/) do |button|
+  click_link_or_button button
+end
+
+When(/^I fill in "([^"]*)" with "([^"]*)"$/) do |arg1, arg2|
+  fill_in(arg1, with: arg2)
+  fill_in(arg1, with: arg2)
+end
+
+Then(/^I click "([^"]*)"$/) do |button|
+  click_link_or_button button
+end
+
+Given(/^I am logged in as "([^"]*)"$/) do |name|
+  user = User.find_by(name: name)
+  login_as(user, scope: :user)
```

Comment:
Good. Glad to see you using the helper methods (remember helpers?) `login_as` is one written by the Devise library.